docs/uperf.md

```diff
@@ -20,6 +20,7 @@ spec:
       serviceip: false
       hostnetwork: false
       pin: false
+      kind: pod
```

Comment:
Could we just default to pod, so we could leave it out of the definition?

Reply:
This is just first iteration, maybe with subsequent pr we can do that?